src/main/menus/app.js

```diff
@@ -146,17 +146,6 @@ function createTemplate(config) {
         },
     }];
 
-    if (process.platform !== 'darwin') {
```

Comment:
shouldn't we just check for linux?

Reply:
Right lol

Reply:
I think we actually want to check for !windows and !macOS, in case someone decides to boot the app up on BSD or sunOS :P

Reply:
yep, I agree. Even if we don't oficially support them, we shouldn't make it harder to run on some OSes :)